src/app/visual/select-field/select-field-visual.component.html

```diff
@@ -51,11 +63,39 @@ <h2>Multiple select mode</h2>
       name="multiple"
       selectMode="multiple"
       [data]="data"
-      [ngModel]="model.multiple"
+      [(ngModel)]="model.multiple"
+      #multipleSelectModel="ngModel"
     >
     </sky-select-field>
   </div>
 
+  <table>
+    <tr>
+      <th>Touched</th>
+      <td>{{ multipleSelectModel.touched }}</td>
+    </tr>
+    <tr>
+      <th>Pristine</th>
+      <td>{{ multipleSelectModel.pristine }}</td>
+    </tr>
+    <tr>
+      <th>Valid</th>
+      <td>{{ multipleSelectModel.valid }}</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+      <td>{{ getJsonValue(multipleSelectModel.value) }}</td>
+    </tr>
+  </table>
+
+  <button
+    class="sky-btn sky-btn-default"
+    type="button"
+    (click)="model.multiple = undefined"
+  >
+    Set ngModel to undefined
+  </button>
+
```

Comment:
☝️ 2 changes to the visual test:
1. Changed `ngModel` to be 2-way bound so setting it to `undefined` would register as a change.
2. Added the nicety of an output table so devs can see how the `ngModel` properties are updated.